README.md

```diff
@@ -37,7 +37,7 @@ state. Some might put form value in a Flux store instead.
         this.state = {formValue: Value(schema, props.value, this.onChange)}
       }
 
-      onChange = (nextFormValue) => {
+      onChange(nextFormValue) {
```

Comment:
I used class property syntax here which is available through babel. Otherwise `onChange` method won't be bound to class instance.